Pokemon Player/headers/Moving.h

```diff
@@ -0,0 +1,24 @@
+#ifndef MOVING_H
+#define MOVING_H
+
+#include <iostream>
+#include <fstream>
+#include "../headers/Control.h"
+
+class Moving {
+
+    public:
+		static void followWall();
```

Comment:
The style guide mandates using two spaces instead of a tab:
http://google-styleguide.googlecode.com/svn/trunk/cppguide.html#Spaces_vs._Tabs
Here's how to configure that behavior on VS:
https://msdn.microsoft.com/en-us/library/ms165330(v=vs.100).aspx